index.html

```diff
@@ -6,8 +6,47 @@
     <title>Send to Whatsapp</title>
     <link href="https://bootswatch.com/flatly/bootstrap.min.css" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/sweetalert/1.1.3/sweetalert.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
```

Comment:
no need to add extra bootstrap css file and jquery. index.html already have bootstrap css file

Reply:
But Your bootstrap css file not supported for navbar.
so be better if you change your bootstrap css to official bootstrap cdn.

Reply:
you can remove my old bootstrap css file and also remove jquery.
i think no need jquery for now. since i will not using dropdown menu.